mage/query-modules/python/kmeans-clustering.md

```diff
@@ -36,25 +36,25 @@ within-the-cluster sum-of-squares. To understand more about it, jump to the [alg
 
 ## Procedures
 
-### `get_clusters( n_clusters, embedding_property, init, n_init, max_iter, tol, algorithm, random_state)`
-This procedure returns for each node to which cluster it belongs to.
+### `get_clusters(n_clusters, embedding_property, init, n_init, max_iter, tol, algorithm, random_state)`
+For each node, this procedure returns what cluster it belongs to.
 
 #### Input:
 
-- `n_clusters : int` ➡ The number of clusters to form.
+- `n_clusters : int` ➡ Number of clusters to be formed.
 - `embedding_property : str` ➡ Node property where embeddings are stored.
-- `init : str` ➡ Initialization method. If `k-means++` is selected, the initial cluster centroids are selected using sampling based on an empirical probability distribution of the points’ contribution to the overall inertia. This technique speeds up convergence and is theoretically proven to be `O(logk)`-optimal. 
-If `random` then `n_clusters` observations (rows) are chosen for the initial centroids randomly from the data.
+- `init : str` ➡ Initialization method. If `k-means++` is selected, initial cluster centroids are sampled per an empirical probability distribution of the points’ contribution to the overall inertia. This technique speeds up convergence and is theoretically proven to be `O(logk)`-optimal. 
+If `random`, `n_clusters` observations (rows) are randomly chosen for the initial centroids.
 - `n_init : int` ➡ Number of times the k-means algorithm will be run with different centroid seeds.
-- `max_iter : int` ➡ Length of one walk in walk sampling.
-- `tol : float` ➡ Relative tolerance with regards to Frobenius norm of the difference in the cluster centers of two consecutive iterations to declare convergence.
-- `algorithm : str` ➡ Options are `lloyd`, `elkan`, `auto`, `full`.
-- `random_state : int` ➡ Use an int to make the randomness deterministic.
+- `max_iter : int` ➡ Length of sampling walks.
+- `tol : float` ➡ Relative tolerance of the Frobenius norm of the difference of cluster centers across consecutive iterations. Used in determining convergence.
+- `algorithm : str` ➡ Options are `lloyd`, `elkan`, `auto`, `full`. Description [here](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#:~:text=algorithm%7B%E2%80%9Clloyd%E2%80%9D%2C%20%E2%80%9Celkan%E2%80%9D%2C%20%E2%80%9Cauto%E2%80%9D%2C%20%E2%80%9Cfull%E2%80%9D%7D%2C%20default%3D%E2%80%9Dlloyd%E2%80%9D).
+- `random_state : int` ➡ Random seed for the algorithm (default:1998).
```

Comment:
Please add the default values from https://github.com/memgraph/mage/blob/main/python/kmeans.py.
Also, make the default value format consistent with existing query modules (e.g. [katz_centrality_online](https://memgraph.com/docs/mage/query-modules/cpp/katz-centrality-online#setalpha-epsilon)) - the values are easy to miss if they’re to the right of the description.

Reply:
Done!